{
    "creator": "Created by epyc, computational experiment management for Python <https://pypi.org/project/epyc/>",
    "version": "2",
    "description": "SEIR simulation with PLC Network (Post)",
    "current": "epyc.resultset.default",
    "locked": false,
    "resultsets": {
        "epyc.resultset.default": {
            "results": [
                {
                    "parameters": {
                        "DN.n": 10000,
                        "epydemic.SEIR.pExposed": 0.018081,
                        "epydemic.SEIR.pInfect": 0.06451835781268038,
                        "epydemic.SEIR.pInfectA": 0.03225917890634019,
                        "epydemic.SEIR.pRemove": 0.1,
                        "epydemic.SEIR.pSymptoms": 0.2,
                        "epydemic.monitor.time_delta": 1
                    },
                    "metadata": {
                        "epyc.experiment.classname": "epydemic.stochasticdynamics.StochasticDynamics",
                        "epyc.experiment.elapsed_time": 0,
                        "epyc.experiment.end_time": "2021-07-24T09:45:57.108643",
                        "epyc.experiment.exception": "'epydemic.generators.PLC.N'",
                        "epyc.experiment.experiment_time": 0.0,
                        "epyc.experiment.rep": 0,
                        "epyc.experiment.repetitions": 5,
                        "epyc.experiment.setup_time": 0.0,
                        "epyc.experiment.start_time": "2021-07-24T09:45:57.106462",
                        "epyc.experiment.status": false,
                        "epyc.experiment.teardown_time": 0.0,
                        "epyc.experiment.traceback": "Traceback (most recent call last):\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epyc/experiment.py\", line 232, in run\n    self.setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkdynamics.py\", line 120, in setUp\n    super().setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkexperiment.py\", line 102, in setUp\n    self._graph = self.networkGenerator().set(params).generate()\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/generator.py\", line 97, in generate\n    return self._generate(self._params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/plc_generator.py\", line 133, in _generate\n    N = params[self.N]\nKeyError: 'epydemic.generators.PLC.N'\n"
                    },
                    "results": {}
                },
                {
                    "parameters": {
                        "DN.n": 10000,
                        "epydemic.SEIR.pExposed": 0.018081,
                        "epydemic.SEIR.pInfect": 0.06451835781268038,
                        "epydemic.SEIR.pInfectA": 0.03225917890634019,
                        "epydemic.SEIR.pRemove": 0.1,
                        "epydemic.SEIR.pSymptoms": 0.2,
                        "epydemic.monitor.time_delta": 1
                    },
                    "metadata": {
                        "epyc.experiment.classname": "epydemic.stochasticdynamics.StochasticDynamics",
                        "epyc.experiment.elapsed_time": 0,
                        "epyc.experiment.end_time": "2021-07-24T09:45:57.108945",
                        "epyc.experiment.exception": "'epydemic.generators.PLC.N'",
                        "epyc.experiment.experiment_time": 0.0,
                        "epyc.experiment.rep": 1,
                        "epyc.experiment.repetitions": 5,
                        "epyc.experiment.setup_time": 0.0,
                        "epyc.experiment.start_time": "2021-07-24T09:45:57.108760",
                        "epyc.experiment.status": false,
                        "epyc.experiment.teardown_time": 0.0,
                        "epyc.experiment.traceback": "Traceback (most recent call last):\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epyc/experiment.py\", line 232, in run\n    self.setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkdynamics.py\", line 120, in setUp\n    super().setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkexperiment.py\", line 102, in setUp\n    self._graph = self.networkGenerator().set(params).generate()\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/generator.py\", line 97, in generate\n    return self._generate(self._params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/plc_generator.py\", line 133, in _generate\n    N = params[self.N]\nKeyError: 'epydemic.generators.PLC.N'\n"
                    },
                    "results": {}
                },
                {
                    "parameters": {
                        "DN.n": 10000,
                        "epydemic.SEIR.pExposed": 0.018081,
                        "epydemic.SEIR.pInfect": 0.06451835781268038,
                        "epydemic.SEIR.pInfectA": 0.03225917890634019,
                        "epydemic.SEIR.pRemove": 0.1,
                        "epydemic.SEIR.pSymptoms": 0.2,
                        "epydemic.monitor.time_delta": 1
                    },
                    "metadata": {
                        "epyc.experiment.classname": "epydemic.stochasticdynamics.StochasticDynamics",
                        "epyc.experiment.elapsed_time": 0,
                        "epyc.experiment.end_time": "2021-07-24T09:45:57.109094",
                        "epyc.experiment.exception": "'epydemic.generators.PLC.N'",
                        "epyc.experiment.experiment_time": 0.0,
                        "epyc.experiment.rep": 2,
                        "epyc.experiment.repetitions": 5,
                        "epyc.experiment.setup_time": 0.0,
                        "epyc.experiment.start_time": "2021-07-24T09:45:57.108978",
                        "epyc.experiment.status": false,
                        "epyc.experiment.teardown_time": 0.0,
                        "epyc.experiment.traceback": "Traceback (most recent call last):\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epyc/experiment.py\", line 232, in run\n    self.setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkdynamics.py\", line 120, in setUp\n    super().setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkexperiment.py\", line 102, in setUp\n    self._graph = self.networkGenerator().set(params).generate()\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/generator.py\", line 97, in generate\n    return self._generate(self._params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/plc_generator.py\", line 133, in _generate\n    N = params[self.N]\nKeyError: 'epydemic.generators.PLC.N'\n"
                    },
                    "results": {}
                },
                {
                    "parameters": {
                        "DN.n": 10000,
                        "epydemic.SEIR.pExposed": 0.018081,
                        "epydemic.SEIR.pInfect": 0.06451835781268038,
                        "epydemic.SEIR.pInfectA": 0.03225917890634019,
                        "epydemic.SEIR.pRemove": 0.1,
                        "epydemic.SEIR.pSymptoms": 0.2,
                        "epydemic.monitor.time_delta": 1
                    },
                    "metadata": {
                        "epyc.experiment.classname": "epydemic.stochasticdynamics.StochasticDynamics",
                        "epyc.experiment.elapsed_time": 0,
                        "epyc.experiment.end_time": "2021-07-24T09:45:57.109209",
                        "epyc.experiment.exception": "'epydemic.generators.PLC.N'",
                        "epyc.experiment.experiment_time": 0.0,
                        "epyc.experiment.rep": 3,
                        "epyc.experiment.repetitions": 5,
                        "epyc.experiment.setup_time": 0.0,
                        "epyc.experiment.start_time": "2021-07-24T09:45:57.109116",
                        "epyc.experiment.status": false,
                        "epyc.experiment.teardown_time": 0.0,
                        "epyc.experiment.traceback": "Traceback (most recent call last):\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epyc/experiment.py\", line 232, in run\n    self.setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkdynamics.py\", line 120, in setUp\n    super().setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkexperiment.py\", line 102, in setUp\n    self._graph = self.networkGenerator().set(params).generate()\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/generator.py\", line 97, in generate\n    return self._generate(self._params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/plc_generator.py\", line 133, in _generate\n    N = params[self.N]\nKeyError: 'epydemic.generators.PLC.N'\n"
                    },
                    "results": {}
                },
                {
                    "parameters": {
                        "DN.n": 10000,
                        "epydemic.SEIR.pExposed": 0.018081,
                        "epydemic.SEIR.pInfect": 0.06451835781268038,
                        "epydemic.SEIR.pInfectA": 0.03225917890634019,
                        "epydemic.SEIR.pRemove": 0.1,
                        "epydemic.SEIR.pSymptoms": 0.2,
                        "epydemic.monitor.time_delta": 1
                    },
                    "metadata": {
                        "epyc.experiment.classname": "epydemic.stochasticdynamics.StochasticDynamics",
                        "epyc.experiment.elapsed_time": 0,
                        "epyc.experiment.end_time": "2021-07-24T09:45:57.109366",
                        "epyc.experiment.exception": "'epydemic.generators.PLC.N'",
                        "epyc.experiment.experiment_time": 0.0,
                        "epyc.experiment.rep": 4,
                        "epyc.experiment.repetitions": 5,
                        "epyc.experiment.setup_time": 0.0,
                        "epyc.experiment.start_time": "2021-07-24T09:45:57.109226",
                        "epyc.experiment.status": false,
                        "epyc.experiment.teardown_time": 0.0,
                        "epyc.experiment.traceback": "Traceback (most recent call last):\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epyc/experiment.py\", line 232, in run\n    self.setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkdynamics.py\", line 120, in setUp\n    super().setUp(params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/networkexperiment.py\", line 102, in setUp\n    self._graph = self.networkGenerator().set(params).generate()\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/generator.py\", line 97, in generate\n    return self._generate(self._params)\n  File \"/home/leopoldpfeiffer/simulations/epi-sim/venv/lib/python3.9/site-packages/epydemic/plc_generator.py\", line 133, in _generate\n    N = params[self.N]\nKeyError: 'epydemic.generators.PLC.N'\n"
                    },
                    "results": {}
                }
            ],
            "pending-results": {},
            "description": "Results collected on 2021-07-24 09:45:57.097665",
            "locked": false
        }
    }
}